openwebstart/src/main/java/com/openwebstart/jvm/ui/dialogs/RuntimeDownloadDialog.java

```diff
@@ -41,7 +42,15 @@ private RuntimeDownloadDialog(final RemoteJavaRuntime remoteRuntime, final Downl
         setTitle(translator.translate("dialog.jvmDownload.title"));
         setResizable(false);
 
-        final JLabel messageLabel = new JLabel(translator.translate("dialog.jvmDownload.message", remoteRuntime.getVersion(), remoteRuntime.getVendor()));
+        // TODO : confirm if we want to display this on label
+        String endpoint = "";
```

Comment:
I do not understand what you do here? should an empty string be displayed in the label? Please remove or refactor

Reply:
next to this the text of `dialog.jvmDownload.message` has not changed. How should this work?